README.md

```diff
@@ -231,7 +231,7 @@ python3 das_cli.py --help
 python3 das_cli.py server --help
 
 # View help for a specific subcommand (e.g., start within the server command)
-python3 das_cli.py server start --help
+python3 das_cli.py db start --help
 
 # View help for the OpenFaaS start command
 python3 das_cli.py faas start --help
```

Comment:
The reason why I explicitly mentioned "and update documentation" in the name of the card is because I wanted to make sure that every single change would be properly documented in the README. But you didn't do that. I see several changes which are not here. Please check every change you made and double check that the README is updated accordingly.